quoridor-app/app/api/gameApi.js

```diff
@@ -0,0 +1,31 @@
+const API_PATH = "http://192.168.86.56:8383/games";
```

Comment:
Couldn't you use an environment variable to define this constant? (https://www.npmjs.com/package/babel-plugin-transform-inline-environment-variables)